TASVideos/Pages/Forum/Subforum/Index.cshtml.cs

```diff
@@ -62,7 +62,7 @@ public async Task<IActionResult> OnGet()
 					CreateTimestamp = ft.CreateTimestamp,
 					Type = ft.Type,
 					PostCount = ft.ForumPosts.Count,
-					LastPost = ft.ForumPosts.Max(fp => (DateTime?)fp.CreateTimestamp)
+					LastPost = ft.ForumPosts.OrderBy(fp => fp.CreateTimestamp).Last()
```

Comment:
I wonder if .OrderByDescending().First() is better.  EF might be smart enough to do the same query either way, but it is worth checking

Reply:
Based on my understanding it is guaranteed at least not to be worse, so I've made the change in the next commit